charters/wg-2021-extension-plan.md

```diff
@@ -20,11 +20,12 @@ During the second 6 months we can begin editor's draft documents for the planned
 | May 18, 2022 | Arch1.1 | Normative feature freeze |
 | mid-May, 2022 | TD1.1 | CR transition |
 | June 3, 2022 | Arch1.1 | CR candidate |
-| mid-June, 2022 | Profiles | Normative feature freeze |
-| June 6-10, 2022 | - | Testfest/Plugfest 2 (focus on Discovery, Profile) |
+| early June, 2022 | Profiles | Normative feature freeze |
```

Comment:
@mmccool I'm curious why the normative feature freeze for WoT Profile has been brought forward from mid-June to early June? That seems unrealistic given the current rate of progress. There are still 20 open issues marked as blocking publication, and lots of normative assertions still to finalise.

Realistically I'd say it's probably going to be delayed by a month or two.

Reply:
@mlagally said it would be possible but I also think that it seems ambitious